lib/livebook_web/live/home_live.ex

```diff
@@ -31,13 +33,7 @@ defmodule LivebookWeb.HomeLive do
     ~H"""
     <div class="flex grow h-full">
       <SidebarHelpers.sidebar>
-        <SidebarHelpers.break_item />
-        <SidebarHelpers.link_item
-          icon="settings-3-fill"
-          label="Settings"
-          path={Routes.settings_path(@socket, :page)}
-          active={false} />
-        <SidebarHelpers.user_item current_user={@current_user} path={Routes.home_path(@socket, :user)} />
+        <SidebarHelpers.shared_home_footer socket={@socket} current_user={@current_user} />
```

Comment:
Maybe `default_footer`, since we mirror it everywhere except the session page?

Reply:
The naming is tricky. I am calling "home" all of the explorer, settings, etc. I think default is not very descriptive. :'( Any other ideas?

Reply:
`shared_home_footer` -> `shared_footer` :see_no_evil:

I cannot think of a good word to describe all off-session pages :<

Reply:
Yeah, I will keep the naming as is, those are all home-like pages or accessible from home. Once we expand things maybe we will have better naming ideas.